### PR TITLE
Create movable-ink-event-streaming.md

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/destinations/movable-ink-event-streaming.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+6. Toggle the **Send events**.
+7. In **Select and filter events** choose which events you want to send. Choose only the events you need in Movable Ink. Transformed events aren't supported.
+8. (optional) In **Select additional properties**, select any more event and user properties you want to send to Movable Ink. If you don't select any properties here, Amplitude doesn't send any. Transformed event properties and transformed user properties aren't supported.
+9. Under **Send Users**, make sure the toggle is enabled ("Users are sent to MovableInk") if you want to stream users and their properties to MovableInk. When enabled, users are automatically created or updated in MovableInk when an event is sent to Amplitude. [Amplitude Identify API](https://www.docs.developers.amplitude.com/analytics/apis/identify-api/) calls are also forwarded to MovableInk. Users aren't sent on a schedule or on-demand using this integration.
+10. (optional) In **Select additional properties**, select any more user properties you want to send to MovableInk. If you don't select any properties here, Amplitude doesn't send any. _Transformed user properties aren't supported._
```
Comment (markzegarelli) [inline docs/data/destinations/movable-ink-event-streaming.md:50]:
```suggestion
10. (optional) In the **Select additional properties** section, select any additional user properties you want to send to Movable Ink. Amplitude sends only the properties you select here. This integration does not support transformed user properties. 
```

File: docs/data/destinations/movable-ink-event-streaming.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+11. When finished, enable the destination and **Save**.
+
+_Once the initial endpoint setup is finalized within Movable Ink and Amplitude, event data will need to be mapped within Movable Ink. Work with your Movable Ink CXM on next-steps._
+
+### Use Cases
+
+- **Generate personalized content:** Generate personalized content relevant to each user using event data from Amplitude.
+- **Analyzing and Segmenting Users:** With Amplitude's analytics capabilities, you can analyze user behavior, identify patterns, and create segments based on specific actions or attributes. For example, you may identify a segment of users who have recently made a purchase or visited a specific page.
```
Comment (markzegarelli) [inline docs/data/destinations/movable-ink-event-streaming.md:58]:
```suggestion
- **Analyze and Segment Users:** With Amplitude's analytics capabilities, you can analyze user behavior, identify patterns, and create segments based on specific actions or attributes. For example, you may identify a segment of users who have recently made a purchase or visited a specific page.
```

File: docs/data/destinations/movable-ink-event-streaming.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- **Sending Event Data to Movable Ink:** Once you have defined your desired segments, you can set up an integration or data pipeline to send the relevant event data from Amplitude to Movable Ink. This could be achieved through APIs, webhook integrations, or other methods supported by both platforms.
```
Comment (markzegarelli) [inline docs/data/destinations/movable-ink-event-streaming.md:59]:
```suggestion
- **Send Event Data to Movable Ink:** After you define your segments, you can configure an integration or data pipeline to send the relevant event data from Amplitude to Movable Ink. You can do this with APIs, web hook integrations, or other methods supported by both platforms.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]
Hi @Brandoncharleskhoo1 Some formatting and verbiage suggestions here.

File: docs/data/destinations/movable-ink-event-streaming.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+- Relevant limits for Movable Ink events are:
+  - Movable Ink does not impose any hard limits on quantity or velocity; however, requests sent too quickly will return 429 responses. Amplitude will handle these automatically. 
+  - The requests must be smaller than 2MB.
+- Amplitude sends select event and user properties along with the event.
+
+## Setup
+
+### Prerequisites
+
+To configure an Event Streaming integration from Amplitude to Movable Ink, you must fulfill the following prerequisites from Movable Ink:
```
[the remark this answers is not in the document]
Comment (Brandoncharleskhoo1) [reply docs/data/destinations/movable-ink-event-streaming.md:26]: Made the changes.

File: docs/data/destinations/movable-ink-event-streaming.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+!!!beta "This feature is in open beta"
+
+    This feature is in open beta and is in active development. Contact the [Movable Ink Client Experience team](https://movableink.com/contact-us) for support with this integration.
+
+[Movable Ink](https:///movableink.com) activates any data into personalized content in any customer engagement and is a powerful extension to the technologies that marketers use today. The platform easily connects to all relevant data—no matter where it lives—updates based upon a recipient's most recent interactions, and is auto-generated by the platform.
+
+## Considerations
+
+- You must enable this integration in each Amplitude project you want to use it in.
+- You need to be an admin of a paid Planhat plan to enable this integration.
```
[the remark this answers is not in the document]
Comment (Brandoncharleskhoo1) [reply docs/data/destinations/movable-ink-event-streaming.md]: Thanks for catching this.

**Pull request merged**